public/app/core/components/AlertBox/AlertBox.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { FunctionComponent, ReactNode } from 'react';
+import classNames from 'classnames';
 import { AppNotificationSeverity } from 'app/types';
 
 interface Props {
   title: string;
   icon?: string;
   body?: ReactNode;
   severity: AppNotificationSeverity;
+  iconOnTop?: boolean;
```

Comment:
do you have really need this? would prefer to have too many variants to show an alert. Test make this the default insteaD?

Reply:
@torkelo In discussions with Diana, we decided to put alert icon on top. So could you explain, do you want to make this option default for all alerts?

![Screenshot from 2019-09-06 15-03-43](https://user-images.githubusercontent.com/4932851/64954356-06a5de00-d88e-11e9-8cc6-29503cf82331.png)

Reply:
yes, think it should be consistent. If we want to show it like this here I think it makes sense to do that everywhere.

Reply:
I feel like we need a little bit more testing before making this default. In some cases it looks inconsistent (alignment at the left and right sides are not the same):
![Screenshot from 2019-09-16 16-36-50](https://user-images.githubusercontent.com/4932851/64962547-4d043880-d8a0-11e9-98bc-32ae78a78e09.png)

Reply:
yea, tested it also. for single line texts it still needs to be centered.

Reply:
So I would rather merge it "as is" and do some investigations in the next iteration.